be/src/storage/inverted/clucene/clucene_inverted_writer.cpp

```diff
@@ -224,6 +235,8 @@ class CLuceneInvertedWriterImpl : public CLuceneInvertedWriter {
     InvertedIndexParserType _parser_type;
     std::wstring _field_name;
     const TabletIndex* _inverted_index;
+    void* _wstring_buffer;
```

Comment:
shall guarantee the buffer will be freed in case of any errors or early exit.

Reply:
Fixed it